docs/installation.rst

```diff
@@ -20,112 +20,137 @@ Installation
+        
+    .. note::
+      Running specific modules or tests will require additional optional dependencies. 
+      The `tardisbase` package is required for running TARDIS Regression Tests.
+      The `viz` package is required for running the TARDIS visualization tools.
+      These optional dependencies can be installed by running:
 
-4. Once finished working, you can deactivate your environment.
-
-  ::
+      .. code-block:: bash
+      
+        $ pip install -e ".[tardisbase,viz]" 
 
-    $ conda deactivate
+      To update optional dependencies, use:
 
-From now on, just activate the ``tardis`` environment before working with the TARDIS package.
+      .. code-block:: bash
+      
+          $ pip install -e ".[tardisbase,viz]" --upgrade --force-reinstall
 
-You have successfully installed TARDIS! 🎉 Please refer to `Quickstart for TARDIS <quickstart.ipynb>`_ 
-to start running simulations.
 
+5. Once finished working, you can deactivate your environment.
 
-.. Install from package
-.. ====================
+  ::
 
-.. It's also possible to install TARDIS by pulling the `conda-forge package <https://anaconda.org/conda-forge/tardis-sn>`_
-.. into a clean environment. However, we still encourage using lockfiles to ensure
-.. reproducibility of scientific results.
+    $ conda deactivate
 
-.. ::
+From now on, just activate the ``tardis-{project_name}`` environment before working with the TARDIS package.
 
-..     $ conda create --name tardis-forge tardis-sn --channel conda-forge
+You have successfully installed TARDIS! 🎉 Please refer to `Quickstart for TARDIS <quickstart.ipynb>`_ 
+to start running simulations.
 
 
 Environment update
 ==================
 
-To update the environment after a new release, download the latest lockfile and run ``conda update``.
+To update the environment, download the latest lockfile and run ``conda update``.
 
-::
+.. code-block:: bash
 
-    $ wget -q https://github.com/tardis-sn/tardis/releases/latest/download/conda-{platform}-64.lock
-    $ conda update --name tardis --file conda-{platform}-64.lock
+    $ wget -q https://github.com/tardis-sn/tardisbase/master/conda-{platform}-64.lock
+    $ conda update --name tardis --file conda-{platform}.lock
 
 .. note::
 
-  If you have installed the tardis package in development mode, you should *ideally* update your environment whenever you pull latest tardis code because the new code added might be using updated (or new) dependencies. If you don't do that and your tardis installation seems broken, you can check if your environment requires update by comparing it against the latest environment file:
+  If you have installed TARDIS in development mode, you should *ideally* update your environment whenever you pull latest code because the new code added might be using updated (or new) dependencies. If you don't do that and your installation seems broken, you can check if your environment requires update by comparing it against the latest environment file:
```

Comment:
```suggestion
  If you have installed TARDIS in development mode, you should *ideally* update your environment whenever you pull the latest code because the new code added might be using updated (or new) dependencies. If you don't do that and your installation seems broken, you can check if your environment requires update by comparing it against the latest environment file:
```